bower.json

```diff
@@ -24,9 +24,19 @@
     "test"
   ],
   "dependencies": {
-    "polymer": "^1.7.0"
+    "polymer": "Polymer/polymer#^1.9.1 || ^2.0.0"
   },
   "devDependencies": {
     "d2l-demo-template": "^0.0.11"
```

Comment:
The latest version (hybrid changes aside) is `0.0.12`.  Did you want to bump to it?  It look like it contains an update to typography.